resources/views/Containers/Components/ContainerItemList.twig

```diff
@@ -6,12 +6,12 @@
     <div>
         <div class="col-xs-12 col-lg-12" v-if="items.length > itemsPerPage">
             <div class="list-item-carousel owl-carousel owl-theme owl-single-item" ref="carouselContainer">
-                <category-item v-for="(item, index) in items" :ref="'categoryItem_' + index" :key="item.data.variation.id" :item-data="item.data" decimal-count="{{ decimalCount | json_encode }}" image-url-accessor="urlMiddle"></category-item>
+                <category-item v-for="(item, index) in items" :ref="'categoryItem_' + index" :key="item.data.variation.id" :item-data="item.data" :decimal-count="{{ decimalCount }}" image-url-accessor="urlMiddle"></category-item>
```

Comment:
`:decimal-count="{{ decimalCount }}"`

ohne das json_encode wird hier bei einem leeren Wert vue.js einen Fehler werfen und eine weiße Seite zeigen.
Der Wert wird momentan noch über `{% set decimalCount = config("Ceres.item.storeSpecial") %}` geladen und nicht durch unser config Objekt (wo der Wert einen default Wert bekommt, falls irgendwas mit der Datenbank nicht stimmt).

Bitte in Zeile 1 noch den Wert auslesen aus `ceresConfig.item.storeSpecial`